fastapi/encoders.py

```diff
@@ -125,10 +125,14 @@ def _jsonable_encoder(
         errors.append(e)
```

Comment:
after this https://gist.github.com/podhmo/adebe386eaf298b5fb7a1689f57f027e#file-03benchmark-txt